docs/tooling/cli-guides/setup-a-devnet.md

```diff
@@ -758,3 +758,144 @@ curl http://52.204.202.216:9650/ext/bc/JjDfmxM3hAEX3VuaKH4PpQskhrvp2pzGTgYLpDwin
 
 C-Chain endpoint follows the same scheme but with the blockchain alias of `C`: `http://52.204.202.216:9650/ext/bc/C/rpc`
 
+<!-- vale off -->
+
+#### Deploy Teleporter
+
+<!-- vale on -->
+
+<!-- vale off -->
+
+[Teleporter](https://github.com/ava-labs/teleporter) is a cross-chain messaging protocol built on top of Warp. Teleporter provides a user-friendly interface to Warp that provides additional features such as replay protection, message delivery incentives, and message execution, to name a few.
+
+Now that the Warp-enabled devnet is up and running, Teleporter can be deployed to the C-Chain and subnet-evm subnets. Teleporter is deployed using [Nick's method](https://yamenmerhi.medium.com/nicks-method-ethereum-keyless-execution-168a6659479c) to ensure that the Teleporter contract is deployed to the same address on all chains. For instructions on how to deploy Teleporter, please see the guide [here.](https://github.com/ava-labs/teleporter/blob/main/utils/contract-deployment/README.md) Follow those instructions to deploy Teleporter to Subnet A, Subnet B, and the C-Chain.
```

Comment:
I believe we need to list here the commands, to deploy the contract(s) and to fund the needed addresses, with a brief explanation of those.

Reply:
I'd prefer to not repeat documentation in multiple places - the requested steps are already included in the linked documentation.